### PR TITLE
Shorten GitHub template and make it more readable

### PR DESCRIPTION
**Pull request by jkmacc-LANL**
This PR moves most of the `ISSUE_TEMPLATE.md` markdown into a new section in `CONTRIBUTING.md`, points the user there, and leaves a shorter, more text-readable version in the template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,38 +1,12 @@
...
-   checking open issues of the corresponding milestone, e.g. see
-
-   * [all open issues in milestone "1.0.x"](
-      https://github.com/obspy/obspy/milestones/1.0.x) or optionally see..
-   * [open and closed issues with "bug" label in that milestone](
-      https://github.com/obspy/obspy/issues?utf8=%E2%9C%93&q=milestone%3A1.0.x+label%3Abug+)).
+Before submitting an Issue, please review the Issue Guidelines (link below).
+
+* Please check whether the bug was already reported or fixed.
+* Please provided the following information:
```
Comment (QuLogic) [inline .github/ISSUE_TEMPLATE.md]: provided -> provide

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,38 +1,12 @@
...
+  -  ObsPy version, Python version and Platform (Windows, OSX, Linux ...)
+  -  How did you install ObsPy and Python (pip, anaconda, from source ...)
+  -  If possible please supply a short, self contained, correct example that
+     demonstrates the issue.
+  -  If this is a regression (used to work in an earlier version of ObsPy),
+     please note where it used to work.
```
Comment (QuLogic) [inline .github/ISSUE_TEMPLATE.md]: where -> when

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,38 +1,12 @@
...
-      which contains bug fixes that will be in the next release) or..
-
- * if it was already reported and/or is maybe even being worked on already by
-   checking open issues of the corresponding milestone, e.g. see
-
-   * [all open issues in milestone "1.0.x"](
-      https://github.com/obspy/obspy/milestones/1.0.x) or optionally see..
-   * [open and closed issues with "bug" label in that milestone](
-      https://github.com/obspy/obspy/issues?utf8=%E2%9C%93&q=milestone%3A1.0.x+label%3Abug+)).
+Before submitting an Issue, please review the Issue Guidelines (link below).
```
Comment (claudiodsf) [inline .github/ISSUE_TEMPLATE.md]: `(link below)` : why not inlining this link?

**Comment by megies** [comment]
Would you mind doing the same refactoring also for the "New PR" page? Would be great!

**Comment by jkmacc-LANL** [comment]
Hehe, sure.  It wasn't until I clicked the PR button that I realized the same could be done for that template :smile:

**Comment by megies** [comment]
Thanks for working on this @jkmacc-LANL! The templates were really too messy (since they are usually only viewed in raw mode).

Good to go, I think.

**Comment by jkmacc-LANL** [comment]
Thanks, my pleasure.

**Pull request merged**